dokan-wpml.php

```diff
@@ -83,6 +89,11 @@ public function __construct() {
 
 		// load all actions and filter under plugins loaded hooks
 	    add_action( 'plugins_loaded', [ $this, 'plugins_loaded' ] );
+
+        // Clear option cache on option updates, additions, deletions
+        add_action( 'updated_option', [ $this, 'clear_option_cache' ], 10, 3 );
```

Comment:
We only use the first parameter here; I think we don't need to call 3 params. We can ignore priority and params if not necessary.